spot-client/src/app.js

```diff
@@ -34,6 +34,9 @@ import {
 import { SpotTVRemoteControlLoader } from './spot-tv/ui/loaders';
 import { SpotTvRestrictedRoute } from './spot-tv/routing';
 
+// This import enables Spot-TV specific analytics
+import 'spot-tv/analytics';
```

Comment:
Import cycle, analytics was importing app-state and this line was importing analytics. Ideally this would be triggered from Spot TV entry point, but it's now shared with remote.